.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+
+python:
+  - 2.7
+
+install:
+  - pip install tox
+script:
+  - tox -e py27
```

Comment:
I wouldn't limit this tox run to the py27 env. Especially since we're going to support more than one travis build in the job. -> #8